src/containers/homeContainer.js

```diff
@@ -21,7 +26,7 @@ class HomeContainer extends Component {
     componentDidMount(){
         console.log("LANGUAGE CONFIG Home -------------------------------------->>>>>>>> ", this.props);
         const EXCHANGETOP = document.getElementById('exchangeInfo').offsetTop * 0.5;
```

Comment:
@ckdbswo Please use Refs from Reactjs. Check this out - https://reactjs.org/docs/refs-and-the-dom.html

Also, Please give your branches more meaningful names for example <Component-name><css-fixes> or something similar to give us at least an idea of what the new branch is about.

Reply:
i did used refs ,changed